### PR TITLE
Mbm 39 myrial steel block

### PR DESCRIPTION
**Pull request by aat79**

### CODE REVIEW

**Review by insertStringHere — APPROVED** [review]

**Comment by MattKirchh** [comment]
Need to add an entry to the `en_us.json` file to give the block a name in-game.

`"block.minebound.myrial_steel_block": "Myrial Steel Block",`

File: src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java [hunk 1]
```diff
@@ -25,6 +25,7 @@ public class BlockRegistry {
     public static final RegistryObject<Block> MYRIAL_CONSOLE = BLOCKS.register("myrial_console", () -> new MyrialConsoleBlock(Properties.copy(Blocks.NETHERITE_BLOCK).lightLevel((i) -> 8)));
     public static final RegistryObject<Block> MYRIAL_GLASS = BLOCKS.register("myrial_glass", () -> new Block(Properties.copy(Blocks.GLASS).noOcclusion().requiresCorrectToolForDrops().strength(30f, 500f)));
 
+    public static final RegistryObject<Block> MYRIAL_STEEL_BLOCK = BLOCKS.register("myrial_steel_block", () -> new Block(Properties.copy(Blocks.NETHERITE_BLOCK)));
```
Comment (MattKirchh) [inline src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java:28]: For the sake of consistency and style, you should remove the empty line before the `MYRIAL_STEEL_BLOCK` declaration and add an empty line below.

Comment (aat79) [reply src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java]: Think I fixed what you meant

**Comment by MattKirchh** [comment]
This block needs to be entered into the `pickaxe.json` file in resources/data/minecraft/tags/blocks/mineable so it can be destroyed.
It also needs to be entered into the `needs_netherite_tool.json` in resources/data/forge/tags/blocks so it can only be mined by a netherite or better pickaxe.
You also need to add a loot table for this block so it will drop itself when destroyed.

You can reference #35 to see these changes.

**Pull request merged**